### PR TITLE
add issue link and author to HPA features

### PR DESCRIPTION
**Pull request by mlycore**
add issue link and author to HPA features

### CODE REVIEW

**Comment by mlycore** [comment]
I signed it!





--

祝好~
苗立尧

在 2017-03-24 09:43:42，"k8s-ci-robot" <notifications@github.com> 写道：


Thanks for your pull request. Before we can look at your pull request, you'll need to sign a Contributor License Agreement (CLA).

📝Please follow instructions at https://github.com/kubernetes/kubernetes/wiki/CLA-FAQ to sign the CLA.

Once you've signed, please reply here (e.g. "I signed it!") and we'll verify. Thanks.

If you've already signed a CLA, it's possible we don't have your GitHub username or you're using a different email address. Check your existing CLA data and verify that your email is set on your git commits.
If you signed the CLA as a corporation, please sign in with your organization's credentials at https://identity.linuxfoundation.org/projects/cncf to be authorized.

Instructions for interacting with me using PR comments are available here. If you have questions or suggestions related to my behavior, please file an issue against the kubernetes/test-infra repository. I understand the commands that are listed here.

File: release-1.6/release-notes-draft.md [hunk 1]
```diff
@@ -39,8 +39,8 @@
 ## Autoscaling
 ### New Features
 ### Notable Changes
-- The Horizontal Pod Autoscaler now supports drawing metrics throug the API server aggregator.
-- The Horizontal Pod Autoscaler now supports scaling on multiple, custom metrics.
+- The Horizontal Pod Autoscaler now supports drawing metrics throug the API server aggregator.([#118](https://github.com/kubernetes/features/issues/118), [@DirectXMan12](https://github.com/DirectXMan12))
```
Comment (liggitt) [inline release-1.6/release-notes-draft.md]: typo " through"

Comment (mlycore) [inline release-1.6/release-notes-draft.md]: sorry .

**Comment by idvoretskyi** [comment]
LGTM

**Pull request merged**